pipeline/metadata/5.2.yaml

```diff
@@ -255,7 +255,7 @@
 - name: "Testing RADOS basic regression scenarios"
   suite: "suites/pacific/rados/tier-2_rados_basic_regression.yaml"
   global-conf: "conf/pacific/rados/7-node-cluster.yaml"
-  platform: "rhel-8"
+  platform: "rhel-9"
   rhbuild: "5.2"
   inventory:
     openstack: "conf/inventory/rhel-8-latest.yaml"
```

Comment:
Platform file needs to be modified to ` rhel-9-latest.yaml `

Reply:
yes.. I will do

Reply:
Updated.